storage/scylla_api.h

```diff
@@ -0,0 +1,199 @@
+  "OORVZT"
+#define ADDRESS_2                                                               \
+  "ADDRESS200000LIKCEJTTIQOTTAWSQCCQQ9A9VOKIWRBWVPXMCGUENWVVMQAMPEIVHEQ9JXLCNZ" \
+  "OORVZT"
+#define ADDRESS_3                                                               \
+  "ADDRESS300000LIKCEJTTIQOTTAWSQCCQQ9A9VOKIWRBWVPXMCGUENWVVMQAMPEIVHEQ9JXLCNZ" \
+  "OORVZT"
+#define ADDRESS_4                                                               \
+  "ADDRESS400000LIKCEJTTIQOTTAWSQCCQQ9A9VOKIWRBWVPXMCGUENWVVMQAMPEIVHEQ9JXLCNZ" \
+  "OORVZT"
+#define ADDRESS_5                                                               \
+  "ADDRESS500000LIKCEJTTIQOTTAWSQCCQQ9A9VOKIWRBWVPXMCGUENWVVMQAMPEIVHEQ9JXLCNZ" \
+  "OORVZT"
+#define ADDRESS_6                                                               \
+  "ADDRESS600000LIKCEJTTIQOTTAWSQCCQQ9A9VOKIWRBWVPXMCGUENWVVMQAMPEIVHEQ9JXLCNZ" \
+  "OORVZT"
+
+#define TRANSACTION_1                                                               \
+  "TRANSACTION100000LIKCEJTTIQOTTAWSQCCQQ9A9VOKIWRBWVPXMCGUENWVVMQAMPEIVHEQ9JXLCNZ" \
+  "OO"
+#define TRANSACTION_2                                                               \
+  "TRANSACTION200000LIKCEJTTIQOTTAWSQCCQQ9A9VOKIWRBWVPXMCGUENWVVMQAMPEIVHEQ9JXLCNZ" \
+  "OO"
+#define TRANSACTION_3                                                               \
+  "TRANSACTION300000LIKCEJTTIQOTTAWSQCCQQ9A9VOKIWRBWVPXMCGUENWVVMQAMPEIVHEQ9JXLCNZ" \
+  "OO"
+#define TRANSACTION_4                                                               \
+  "TRANSACTION400000LIKCEJTTIQOTTAWSQCCQQ9A9VOKIWRBWVPXMCGUENWVVMQAMPEIVHEQ9JXLCNZ" \
+  "OO"
+#define TRANSACTION_5                                                               \
+  "TRANSACTION500000LIKCEJTTIQOTTAWSQCCQQ9A9VOKIWRBWVPXMCGUENWVVMQAMPEIVHEQ9JXLCNZ" \
+  "OO"
+#define TRANSACTION_6                                                               \
+  "TRANSACTION600000LIKCEJTTIQOTTAWSQCCQQ9A9VOKIWRBWVPXMCGUENWVVMQAMPEIVHEQ9JXLCNZ" \
+  "OO"
+#define TRANSACTION_7                                                               \
+  "TRANSACTION700000LIKCEJTTIQOTTAWSQCCQQ9A9VOKIWRBWVPXMCGUENWVVMQAMPEIVHEQ9JXLCNZ" \
+  "OO"
+#define TRANSACTION_8                                                               \
+  "TRANSACTION800000LIKCEJTTIQOTTAWSQCCQQ9A9VOKIWRBWVPXMCGUENWVVMQAMPEIVHEQ9JXLCNZ" \
+  "OO"
+#define TRANSACTION_9                                                               \
+  "TRANSACTION900000LIKCEJTTIQOTTAWSQCCQQ9A9VOKIWRBWVPXMCGUENWVVMQAMPEIVHEQ9JXLCNZ" \
+  "OO"
+
+#define TIMESTAMP_1 1
+#define TIMESTAMP_2 2
+#define TIMESTAMP_3 3
+
+#define MESSAGE_1 "MESSAGE1"
+#define MESSAGE_2 "MESSAGE2"
+#define MESSAGE_3 "MESSAGE3"
+#define MESSAGE_4 "MESSAGE4"
+#define MESSAGE_5 "MESSAGE5"
+#define MESSAGE_6 "MESSAGE6"
+
+typedef struct scylla_iota_transaction scylla_iota_transaction_t;
+
+status_t new_scylla_iota_transaction(scylla_iota_transaction_t** obj);
```

Comment:
These two functions are only used in tests. Are they functions especially for testing? If so, move them to `/test`